bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -315,4 +316,9 @@ default String getRedirectUrl() {
     default String getPrefillService() {
         return null;
     }
+
+    @JsonIgnore
+    default String getContextPath() {
+        return StringUtils.EMPTY;
```

Comment:
pls dont add this at interface level. contextpath for formcontainer does not makes sense. it belongs to request.
test for the FormContainer props which use this context path